.circleci/file_size_check.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+EXIT_CODE=0
+for SIZE in $(git log --pretty='format:%H' master.. | git cat-file --batch-check='%(objectsize)')
```

Comment:
Regardless of the other problems this is not working I think, it seems to reflect the size of the commit (like message size?) but not the contained objects (like binary files commited). My oneliner was just food for thought, it might need digging further into the commit/object structure of git like checking the children of the commit object or something..

Something like this maybe https://askubuntu.com/questions/618186/getting-commit-sizes-in-git